networking_nsxv3/common/scheduling.py

```diff
@@ -44,8 +44,8 @@ def __enter__(self):
             sleeptime = run_time - self.schedule[offset] + self.limit
             if self.callback:
                 eventlet.spawn(self.callback, sleeptime)
-            eventlet.greenthread.sleep(sleeptime)
             run_time = self.schedule[offset] + self.limit
```

Comment:
`eventlet.spawn()` calls into eventlet, too, so there might be greenthread-scheduling involved. Could we move the `run_time` computation 2 more lines up?